avionics/common/src/roar/buffer.cpp

```diff
@@ -0,0 +1,238 @@
+#include <cassert>
```

Comment:
What happens if these asserts are hit running on the board?

Reply:
There's an option to have asserts compiled out under release mode - so that's the plan. Will make a ticket to ensure that this actually happens.

Alternatively, convert asserts to a log call on assertion fail.